pkg/webui/console/views/application-collaborator-edit/index.js

```diff
@@ -72,7 +72,7 @@ import api from '../../api'
     dispatch(getApplicationCollaborator(appId, collaboratorId, isUser))
   },
   redirectToList (appId) {
-    dispatch(replace(`/console/applications/${appId}/collaborators`))
+    dispatch(replace(`/applications/${appId}/collaborators`))
```

Comment:
since you are doing this to other files, move this to `matchDispatchToProps`

Reply:
This is already in map dispatch to props.